### PR TITLE
Update /learning

### PR DESCRIPTION
**Pull request by vcarl**
Thanks for sharing the content @markerikson !

Our `/learning` page is 5 years old, and hasn't seen a substantial update since it was first written. Mark very helpfully provided a huge amount of resources that he wrote for his internal team, but there's room to improve it further! Leaving this PR open for a few days for more discussion.

### CODE REVIEW

File: src/md-pages/learning.md [hunk 1]
```diff
@@ -5,272 +5,300 @@ sidebar: true
...
+- CSS and layout
+  - [CSS: From Zero to Hero](https://dev.to/aspittel/css-from-zero-to-hero-3o16)
+  - [MDN - Visual Formatting Model](https://developer.mozilla.org/en-US/docs/Web/CSS/Visual_formatting_model)
+  - [MDN - Introduction to the CSS Box Model](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model)
+  - [HTML and CSS is Hard (but it doesn't have to be)](https://internetingishard.com/html-and-css/)
+  - [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
+  - [A Complete Guide to Grid](https://css-tricks.com/snippets/css/complete-guide-grid/)
+- Node / NPM
+  - [How to Install and Use Node.js and NPM](https://www.taniarascia.com/how-to-install-and-use-node-js-and-npm-mac-and-windows/)
+  - [Node.js Tutorial - Step by Step Guide for Getting Started](https://www.airpair.com/javascript/node-js-tutorial)
```
Comment (nickserv) [inline src/md-pages/learning.md]:
```suggestion
  - [How to install Node.js](https://nodejs.dev/learn/how-to-install-nodejs)
  - [An introduction to the npm package manager
](https://nodejs.dev/learn/an-introduction-to-the-npm-package-manager)
```

File: src/md-pages/learning.md [hunk 2]
```diff
@@ -5,272 +5,300 @@ sidebar: true
...
+  - [A Guide to JavaScript Regular Expressions](https://flaviocopes.com/javascript-regular-expressions/)
+  - [A Beginner's Guide to Regular Expressions in JavaScript](https://blog.bitsrc.io/a-beginners-guide-to-regular-expressions-regex-in-javascript-9c58feb27eb4)
+- CSS
+  - [Online Interactive CSS Cheat Sheet](https://htmlcheatsheet.com/css/)
+  - [A Practical CSS Cheat Sheet](https://www.toptal.com/css/css-cheat-sheet)
+  - [GRID: A simple visual cheatsheet for CSS Grid Layout](http://grid.malven.co/)
+- Node / NPM
+  - [TutorialsPoint - Node.js Tutorial](https://www.tutorialspoint.com/nodejs/index.htm)
+  - [The `package.json` Guide](https://flaviocopes.com/package-json/)
+  - [Anatomy of a `package.json` File](https://alligator.io/nodejs/package-json/)
```
Comment (nickserv) [inline src/md-pages/learning.md]:
```suggestion
  - [Introduction to Node.js](https://nodejs.dev/learn)
  - [The package.json guide](https://nodejs.dev/learn/the-package-json-guide)
```

**Review by nickserv — COMMENTED** [review]
Here are some suggestions for links pointing to `nodejs.dev`, which has the new Node.js community tutorials

**Comment by ghost** [comment]
@vcarl sir add [this tut](https://javascript.info) also in js section. I have learnt js from there only

**Comment by markerikson** [comment]
@Padmanabh82 good news - it's in there already :)

https://github.com/reactiflux/reactiflux.com/blob/vc-update-learning/src/md-pages/learning.md?plain=1#L46

**Pull request merged**